.github/workflows/auto-label-issues.yaml

```diff
@@ -0,0 +1,17 @@
+name: Label issues
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - run: gh issue edit "$NUMBER" --add-label "$LABELS"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this still a valid approach? The other actions normally use this bot-app to retrieve a token...

Reply:
The plain github token is enough for this process with the right permission. There is no need for the bot account level token.